torch_em/trainer/wandb_logger.py

```diff
@@ -11,16 +13,27 @@
 
 class WandbLogger:
     def __init__(self, trainer):
+        self.log_dir = f"./logs"
```

Comment:
I wouldn't use `f""` strings if there's nothing to insert (my linter would complain about this.)

Reply:
On the other hand, should this be f"./logs/{trainer.name}", or does wandb handle this already internally?

Reply:
I think wandb handles this internally... these files are semi temporary anyway (or downloads of artefacts). the log is really just valuable/valid if uploaded to a server (this server might run locally in docker though)
because
```python
run = wandb.init(name=None, dir="logs")
 name = run.name
 ```
 
 I left out the name in the log dir (the default logdir is `./wandb` and that works just fine...

Reply:
> I think wandb handles this internally... these files are semi temporary anyway (or downloads of artefacts). the log is really just valuable/valid if uploaded to a server (this server might run locally in docker though)

Ok, makes sense.